styles.css

```diff
@@ -0,0 +1,841 @@
+q:before, q:after {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.header {
+  background-image: url(./src/images/photos/header.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding: 28px 20px 40px;
+  min-height: 100vh;
+  box-sizing: border-box;
+  grid-template-rows: 1fr 1fr;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(2, 130px);
+  gap: 20px;
+}
+@media (min-width: 744px) {
+  .header {
+    grid-template-columns: repeat(6, 80px);
+    gap: 24px;
+  }
+}
+@media (min-width: 1260px) {
+  .header {
+    grid-template-columns: repeat(12, 63px);
+    gap: 24px;
+  }
+}
+.header__navigation {
+  grid-column: span 2;
+  grid-row: 1/2;
+  align-self: start;
+}
+@media (min-width: 744px) {
+  .header__navigation {
+    grid-column: span 6;
+  }
+}
+@media (min-width: 1260px) {
+  .header__navigation {
+    grid-column: span 12;
+  }
+}
+.header__title {
+  grid-column: 1/3;
+  grid-row: 2/3;
+  align-self: end;
+}
+@media (min-width: 744px) {
+  .header__title {
+    grid-column: span 6;
+  }
+}
+@media (min-width: 1260px) {
+  .header__title {
+    grid-column: span 12;
+  }
+}
+
+.title {
+  font-family: "Inter";
+  font-style: italic;
+  font-weight: 800;
+  font-size: 32px;
+  line-height: 32px;
+  letter-spacing: -1.5px;
+  color: #292929;
+  position: relative;
+}
+@media (min-width: 744px) {
+  .title {
+    font-size: 40px;
+    line-height: 40px;
+  }
+}
+.title--header::before {
+  content: "";
+  background-image: url(../src/images/icon-sound-waves.svg);
+  background-repeat: no-repeat;
+  width: 48px;
+  height: 58px;
+  position: absolute;
+  bottom: 80px;
+  left: 0px;
+  background-size: contain;
+}
+@media (min-width: 744px) {
+  .title--header::before {
+    bottom: 110px;
+    width: 63px;
+    height: 76px;
+  }
+}
+.title--secondary {
+  font-style: normal;
+  font-weight: 700;
+}
+@media (min-width: 744px) {
+  .title--secondary {
+    font-size: 40px;
+    line-height: 40px;
+  }
+}
+@media (min-width: 1260px) {
+  .title--secondary {
+    font-size: 48px;
+    line-height: 48px;
+  }
+}
+
+.logo__image {
+  height: 16px;
+}
+@media (min-width: 744px) {
+  .logo__image {
+    height: 24px;
+  }
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+}
+.navigation__item {
+  display: flex;
+  align-items: center;
+}
+.navigation__item--hidden {
+  opacity: 0;
+}
+.navigation__icons {
+  display: flex;
+  align-items: center;
+}
+
+.burger::before {
+  content: "";
+  background-image: url(./src/images/icon-burger.svg);
+  background-size: cover;
+  display: inline-block;
+  width: 18px;
+  height: 12px;
+}
+@media (min-width: 744px) {
+  .burger::before {
+    width: 24px;
+    height: 16px;
+  }
+}
+.burger::after {
+  display: none;
+  content: "";
+  background-image: url(./src/images/icon-burger-hover.svg);
```

Comment:
As with the previous point, consider using a relative path for your background image.